apis/python/src/tiledb/vector_search/storage_formats.py

```diff
@@ -0,0 +1,18 @@
+storage_formats = {
+    "0.1": {
+        "CENTROIDS_ARRAY_NAME": "centroids.tdb",
+        "INDEX_ARRAY_NAME": "index.tdb",
+        "IDS_ARRAY_NAME": "ids.tdb",
+        "PARTS_ARRAY_NAME": "parts.tdb",
+        "PARTIAL_WRITE_ARRAY_DIR": "write_temp",
+    },
+    "0.2": {
```

Comment:
It's a little confusing that we are adding this here but setting the default to 0.1

Reply:
eg https://github.com/TileDB-Inc/TileDB-Vector-Search/pull/118/files#diff-b0ba694c23a77c9684b7d183ae060fd615e208a54123ed3d384a3aaf9d30b214R51

Reply:
nvm I see, that is intended as the "fallback" version where unspecified in the file -- which is for all existing files -- and we are bumping the format to use 0.2 names for all new ingestions. 👍